_includes/tutorials/kafka-connect-datagen-ccloud/kafka/markup/dev/provision-connector.adoc

```diff
@@ -0,0 +1,6 @@
+Provision the Kafka Connect Datagen source connector with the following command.
+Notice that it references the connector configuration file `datagen-source-config.json` that you created in the previous step.
```

Comment:
FWIW I got tripped up here because my "current" kafka cluster was not set to the proper one I had created the key for.  Maybe we need to add an explicit `ccloud kafka cluster use <your-cluster-id>` in a previous step to ensure the user gets that (even though it might be implied in some linked instructions above).   Additionally, we have them delete resources in the cleanup section so this might help avoid unintended deletions for students.

Reply:
If we go the `ccloud-stack` route, this will no longer be an issue.